modules/addons/transports/udp/src/main/java/org/jboss/wsf/stack/cxf/addons/transports/udp/UDPConduit.java

```diff
@@ -165,6 +165,7 @@ public void close() throws IOException
                   catch (Exception e)
                   {
                      //ignore
+                     LOG.warning(e.getMessage());
```

Comment:
LOG.trace(e) should be the solution I think.

You are omitting stacktrace by calling just e.getMessage() and trace is sufficient level